src/ui/CustomCollapsible/CustomCollapsible.svelte

```diff
@@ -0,0 +1,110 @@
+<script>
+  export let title = "";
```

Comment:
I think as a name, `CustomCollapsible` doesn't really tell you much about what it _does_, just that you've changed it a bit. This was made to allow nested collapsibles, right? Is `NestableCollapsible` a better name?

Reply:
Done, I renamed the component 👍